### DISCUSSION TITLE
SET GLOBAL local_infile = true with estuary

### DISCUSSION THREAD

**Discussion by itos-brian** [planetscale/discussion#525] [Q&A] [answered]
Hi all, I was trying to import data into a planetscale db with estuary. It seems like we need to run something like 
`SET GLOBAL local_infile = true` to get the import process to work. Is this possible with the current setup?

`ERRO
[0003] the materialization cannot run due to the following error(s):
 - `local_infile` global variable must be enabled on your mysql server. You can enable this using `SET GLOBAL local_infile = true`.  
source
="<nil>"
ERRO
[0003] connector failed                              
module
="connector_init::rpc" 
status
="exit status: 1"
WARN
[0003] connector exited with unconsumed input stream remainder  
module
="connector_init::rpc"
flow.json error at /materializations/Itos~1onchain~1materialize-mysql :
the materialization cannot run due to the following error(s):
 - `local_infile` global variable must be enabled on your mysql server. You can enable this using `SET GLOBAL local_infile = true`.
FATA
[0003] fatal error   `

**Comment by orware** [comment, marked as answer]
Hello @itos-brian,

This isn't currently [supported within Vitess](https://github.com/vitessio/vitess/issues/2976) and based on my limited knowledge of [Estuary](https://estuary.dev/), my guess is that it would likely require a custom connector, specific to PlanetScale, in order to work around the more MySQL-specific approach they are currently utilizing.

Our PlanetScale Connect feature provides the general functionality needed for tools like Estuary to do this and we have even built a few of these integrations ourselves with the [Airbyte](https://planetscale.com/docs/integrations/airbyte) and [Stitch](https://planetscale.com/docs/integrations/stitch) integrations, but more would need to be built for other similar systems such as Estuary.
